SourceCode/raytracer/Triangle.h

```diff
@@ -0,0 +1,160 @@
+#pragma once
+#include "CRTTypes.h"
+
+// hw4 Task 1
+class Triangle {
+public:
+    Vec3 v0, v1, v2;
```

Comment:
It's often necessary to be able to index into the vertices, so storing them as array could be more convenient.